src/lodash-modules.js

```diff
@@ -1,35 +1,28 @@
 import path from 'path';
-import {includes, result, zipObject} from 'lodash';
+import {findKey, transform} from 'lodash';
 import fs from 'fs';
-import {aliasToReal} from 'lodash-cli/lib/mapping';
```

Comment:
Ah. From my end lodash provides the alias modules so no mapping needs to be done.